wallet/udb/txmined.go

```diff
@@ -1555,18 +1490,12 @@ func (s *Store) AddCredit(ns walletdb.ReadWriteBucket, rec *TxRecord, block *Blo
 			Put(k, v)
 		if err != nil {
 			const str = "failed to write invalidated credit"
-			return storeError(ErrDatabase, str, err)
+			return storeError(apperrors.ErrDatabase, str, err)
 		}
 		return nil
 	}
 
-	isNew, err := s.addCredit(ns, rec, block, index, change, account)
-	if err == nil && isNew && s.NotifyUnspent != nil {
-		// This causes a lockup because wtxmgr is non-reentrant.
-		// TODO: move this call outside of wtxmgr and do not use
-		// a passthrough to perform the notification.
-		// s.NotifyUnspent(&rec.Hash, index)
-	}
+	_, err := s.addCredit(ns, rec, block, index, change, account)
```

Comment:
Since it has been broken for quite some time, and I don't see any issue about it, I'll open a new one